eng/templates/stages/deploy.yaml

```diff
@@ -203,6 +203,24 @@ stages:
         .\darc\darc.exe get-default-channels --source-repo arcade-services --ci -t "$(GetAuthInfo.FederatedToken)" --bar-uri "$(GetAuthInfo.BarUri)" --debug
       displayName: Test Federated token authentication
 
+    - ${{ if in(variables['Build.SourceBranch'], 'refs/heads/main', 'refs/heads/production') }}:
+      - powershell: 
+          $darcBuild = .\darc\darc.exe get-build `
+            --repo "https://github.com/dotnet/arcade-services" `
+            --commit $(Build.SourceVersion) `
+            --ci `
+            -t "$(GetAuthInfo.FederatedToken)" `
```

Comment:
You could probably do the same as the tests below and use the JWT token directly instead of the federated one. So `-p $(GetAuthInfo.Token)`

Reply:
Same in the command below

Reply:
updated